CMake/External_OpenCV_contrib.cmake

```diff
@@ -0,0 +1,17 @@
+# The OpenCV contrib repo external project
```

Comment:
This approach is ok, but I think it might be cleaner to add the contents of this file to the External_OpenCV.cmake and run it as a custom step after the OpenCV build is done, e.g. 
  if (BUILD_OpenCV_CONTRIB)
    ExternalProject_Add_Step(OpenCV build_contrib
      COMMENT "Build the OpenCV contrib modules"
      DEPENDEES build
      ........
      )
  endif()
   
or something like that.